src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -47,6 +50,21 @@ public AddCommand(Person person) {
         toAdd = person;
     }
 
+    /**
+     * Gets the warning message for the command's execution.
+     *
+     * @return The warning message, or an empty string if none.
+     */
+    private String getMessageWarn() {
+        boolean isPhoneOfExpectedFormat = toAdd.getPhone().isExpectedFormat();
+
+        if (!isPhoneOfExpectedFormat) {
+            return String.format(MESSAGE_WARN, Phone.MESSAGE_EXPECTED);
+        }
+
+        return "";
+    }
```

Comment:
Would moving this method to the `Phone` class make sense as it's a function returning a message that only has to do with `Phone` objects?

It would also reduce code duplication as the same method is defined in `EditCommand`.

Reply:
Already did create a method in the phone class to indicate if the phone number is of an expected format.

This method generates the warning message as part of the AddCommand's output, hence I'd thought it be more appropriate to add it within this class. Different commands may produce different warning messages, it may be duplicated code now it might be possible that we wish add a warning in AddCommand that may not be present in the EditCommand.

Reply:
There also needs to be a place within the AddCommand class to add the warning message, rather than doing it in the `execute` method, I'd thought it be better to extract it  out.